forms-flow-api/src/formsflow_api/models/employee_data.py

```diff
@@ -18,6 +23,12 @@ def __init__(self, data):
    self.empId = data.get("EMPLID")
    self.positionTitle = data.get("position_title")
    self.depId = data.get("DEPTID")
+   self.officeAddress = ", ".join(filter(None, 
+                                  (data.get("office_address1").strip(),
+                                   data.get("office_address2").strip(),
+                                   data.get("office_city").strip(),
+                                   data.get("office_stateprovince").strip(),
+                                   data.get("office_postal").strip())))
```

Comment:
I'd improve this code to ensure `strip()` run only on string, in cases one of the office_* attributes were None
```suggestion
self.officeAddress = ", ".join(filter(None, [
    value.strip() for key, value in [
        ("office_address1", data.get("office_address1")),
        ("office_address2", data.get("office_address2")),
        ("office_city", data.get("office_city")),
        ("office_stateprovince", data.get("office_stateprovince")),
        ("office_postal", data.get("office_postal"))
    ] if value is not None and isinstance(value, str)
]))
```